lib/helper/item-route-validators.js

```diff
@@ -0,0 +1,196 @@
+const validator = require('validator');
```

Comment:
Just wonder if this file should be called `item-route-validators` or something a bit more generic, since we may want to use validators from here in other places?

Reply:
@batpad while I was writing I thought having a route specific validator would be good. 
We can totally iterate on this in future.